src/config.cpp

```diff
@@ -105,6 +105,15 @@ void loadConfigFromJsonObject(JsonDocument &doc) {
 
   // Copy Vibration Settings
   Config.vibration_mode = (VibrationMode)(doc["vibration_mode"] | (int)VibrationMode::RampStop);
+  
+  // Post orgasm torture settings. The clench_pressure_sensitivity might need ajustment on the Edge-o-matic
+  Config.clench_pressure_sensitivity = doc["clench_pressure_sensitivity"] | 200;
+  Config.clench_threshold_2_orgasm = doc["clench_threshold_2_orgasm"] | 35;
+  Config.clench_detector_in_edging = doc["clench_detector_in_edging"] | false;
+  Config.auto_edging_duration_minutes = doc["auto_edging_duration_minutes"] | 30;
+  Config.post_orgasm_duration_seconds = doc["post_orgasm_duration_seconds"] | 10;
+  Config.post_orgasm_menu_lock = doc["post_orgasm_menu_lock"] | false;
+  Config.edge_menu_lock = doc["edge_menu_lock"] | false;
```

Comment:
I don't see an initializer for the max_clench_duration noted above?

Is `edge_menu_lock` supposed to be something we persist after reboot? It seems like a transient state that's just supposed to have meaning during runtime. Generally, Config shouldn't hold state items, but I understand the convenience of a global structure.

If it wasn't meant to persist and it'd be a complicated change to refactor, go ahead and file an issue with a note to take care of it later, and I'll get on it at some point.

Reply:
edge_menu_lock is a state achieved past a number of time to not lock out the user from modifying the edging thresholds

Reply:
edge_menu_lock and port_orgasm_menu_lock are not transient. they state at with moment in the session the menu gets locked out.  If you turn on Edge_menu_lock, the lock occurs after 2 minutes in the edging session and stays locked until end of the session. if you turn on post_orgasm_menu_lock and turn of edge_meu_lock then the lock occurs as soon as the orgasm is detected and stays locked until post orgasm timer runs out

Reply:
Excellent, and thank you for the clarification. I interpreted this to be a state, but now it makes sense.